### DISCUSSION TITLE
Display Live Streaming Data

### DISCUSSION THREAD

**Discussion by JSNN17** [TomSchimansky/CustomTkinter#526] [General]
Hi,

I was wondering if it was possible to display live on the GUI. Like import data from an excel file and  display the data over time.

Thanks

**Comment by Akascape** [comment]
@JSNN17 Possible ig, but you may have to add some original tkinter widgets with customtkinter. (Eg. table/treeviews etc which are not currently available in customtkinter)
Tkinter widgets can be customized and implemeted in customtkinter window easily. Like this is an example of treeview: https://github.com/TomSchimansky/CustomTkinter/discussions/524#discussioncomment-3898307

**I would recommend you to first make the program in tkinter, if it is working then port it to customtkinter.**

**Reply by JSNN17** [reply]
Thanks for the advice!
